packages/notify-slack/README.md

```diff
@@ -0,0 +1,31 @@
+# `notify-slack` - **Github Action**
+
+This action will send a slack message
+
+## Input
+
+| Name          | Description                                             |
+| ------------- |---------------------------------------------------------|
+| `CHANNEL_ID` | The slack channel id                                     |
+| `MESSAGE`       | The message display in first block of notification    |
+| `SLACK_BOT_TOKEN`       | OAuth token of your slack app                 |
+
+## Example Workflow File
+
+```yaml
+name: Notify slack
+
+on: [pull_request]
+
+jobs:
+    notify-slack:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Send message to Slack channel
+            if: ${{ failure() }}
+            uses: bonitasoft/actions/packages/notify-slack@main
+            with:
+              CHANNEL_ID: ${{ secrets.SLACK_UID_CHANNEL_ID }}
+              MESSAGE: ":x: Build on branch ${{github.ref}} failed"
+              SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
```

Comment:
```suggestion
              SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
              TRIGGER_CONDITION: ${{ failure() }} // or ${{ always() }}...
```

Reply:
We don't need this trigger condition, you can already use `if` keyword in called step